content/en/agent/basic_agent_usage/windows.md

```diff
@@ -99,7 +100,7 @@ Each configuration item is added as a property to the command line. The followin
 | `DDAGENTUSER_PASSWORD`                      | String  | Override the cryptographically secure password generated for the `ddagentuser` user during Agent installation _(v6.11.0+)_. Must be provided for installs on domain servers. [Learn more about the Datadog Windows Agent User][3].  |
 | `APPLICATIONDATADIRECTORY`                  | Path    | Override the directory to use for the configuration file directory tree. May only be provided on initial install; not valid for upgrades. Default: `C:\ProgramData\Datadog`. _(v6.11.0+)_                                           |
 | `PROJECTLOCATION`                           | Path    | Override the directory to use for the binary file directory tree. May only be provided on initial install; not valid for upgrades. Default: `%ProgramFiles%\Datadog\Datadog Agent`. _(v6.11.0+)_                                    |
-| `ADDLOCAL`                                  | String  | Enable additional agent component. Setting to `"MainApplication,NPM"` causes the driver component for [Network Performance Monitoring][4] to be installed.                                                                          |
+| `ADDLOCAL`                                  | String  | Enable additional agent component. Setting to `"MainApplication,NPM"` causes the driver component for [Network Performance Monitoring][4] to be installed. _(<= v7.44.0)_                                                           |
```

Comment:
```suggestion
| `ADDLOCAL`                                  | String  | Enable additional agent component. Setting to `"MainApplication,NPM"` causes the driver component for [Network Performance Monitoring][4] to be installed. _(v7.44.0 or previous)_                                                           |
```

Reply:
Should we state that this is "legacy" from 7,45 and up. Sure v7.44.0 or previous implies it but it is weird especially when all other options use xxx+.

Reply:
Thanks for pointing that out, it makes sense to me. In the latest review I'm suggesting to move this content to a `Legacy configuration` table as you mentioned in your other [comment](https://github.com/DataDog/documentation/pull/17951/files#r1206040488), but happy to keep iterating on that if you have any other suggestions.